Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -496,7 +498,7 @@
   - PRETEND TO BE DRUNK
   - QUESTION AUTHORITY
   - QUOTE PEOPLE
-  - RAP
+  - QUOTE SHAKESPEARE
```

Comment:
50/50 on this. I cringe at bad rap attempts rather than laugh but it's personal humour, I think? Thoughts?